notebooks/week5/02_experiment_with_model_eval.py

```
@@ -0,0 +1,154 @@
+
+print(f"MAE for New Model: {mae_new}")
+print(f"MAE for Old Model: {mae_old}")
+
+# COMMAND ----------
+
+if mae_new < mae_old:
+    print("New model is better based on MAE.")
+    model_version = mlflow.register_model(
+        model_uri=new_model_uri,
+        name=f"{catalog_name}.{schema_name}.house_prices_model_fe",
+        tags={"git_sha": f"{git_sha}",
```

Ruff (0.7.0)

Undefined name `git_sha`

(F821)